src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
```

Comment:
As I see this is just `user` endpoint method. So this file has to be named `src/api/users`.
So it will be consistent with all other API modules.